src/compiler-debugging.md

```diff
@@ -207,10 +207,10 @@ $ RUSTDOC_LOG=rustdoc rustdoc +local my-file.rs
 
 While calls to `error!`, `warn!` and `info!` are included in every build of the compiler,
 calls to `debug!` and `trace!` are only included in the program if
-`debug-assertions=yes` is turned on in config.toml (it is
+`debug-assertions=true` is turned on in config.toml (it is
```

Comment:
Good catch, thanks! Although this actually changed recently to be more specific: https://github.com/rust-lang/rust/pull/76588
```suggestion
`debug-logging=true` is turned on in config.toml (it is
```